components/shared/workCard/MaterialTypeIconWrapper.tsx

```diff
@@ -1,21 +1,25 @@
 import React from "react"
 
+import Icon from "@/components/shared/icon/Icon"
 import { cn } from "@/lib/helpers/helper.cn"
 
-import Icon from "../icon/Icon"
+export type MaterialTypeIconWrapperProps = {
+  iconName: "ebook" | "book" | "headphones" | "controller" | "video" | "podcast"
```

Comment:
I think we should move this to our global type definitions in the global lib folder